tools/restatectl/src/commands/metadata/patch.rs

```diff
@@ -59,57 +57,20 @@ pub(crate) async fn patch_value(
     let patch = serde_json::from_str(opts.patch.as_str())
         .map_err(|e| anyhow::anyhow!("Parsing JSON patch: {}", e))?;
 
-    let value = match opts.metadata.access_mode {
-        MetadataAccessMode::Remote => patch_value_remote(connection, opts, patch).await?,
-        MetadataAccessMode::Direct => patch_value_direct(opts, patch).await?,
-    };
+    let value = patch_value_inner(connection, opts, patch).await?;
 
     let value = serde_json::to_string_pretty(&value).map_err(|e| anyhow::anyhow!(e))?;
     println!("{value}");
 
     Ok(())
 }
 
-async fn patch_value_remote(
+async fn patch_value_inner(
     connection: &ConnectionInfo,
     opts: &PatchValueOpts,
     patch: Patch,
-) -> anyhow::Result<Option<GenericMetadataValue>> {
-    let metadata_store_client = create_metadata_store_client(connection, &opts.metadata).await?;
-    Ok(Some(
-        patch_value_inner(opts, &patch, &metadata_store_client).await?,
-    ))
-}
-
-async fn patch_value_direct(
-    opts: &PatchValueOpts,
-    patch: Patch,
-) -> anyhow::Result<Option<GenericMetadataValue>> {
-    let value = run_in_task_center(opts.metadata.config_file.as_ref(), |config| async move {
-        let rocksdb_manager = RocksDbManager::init(Configuration::mapped_updateable(|c| &c.common));
-        debug!("RocksDB Initialized");
-
-        let metadata_store_client = start_metadata_server(config.clone()).await?;
-        debug!("Metadata store client created");
-
-        let result = patch_value_inner(opts, &patch, &metadata_store_client).await;
-
-        rocksdb_manager.shutdown().await;
-        result
-    })
-    .await?;
-
-    Ok(Some(value))
-}
-
-async fn patch_value_inner(
-    opts: &PatchValueOpts,
-    patch: &Patch,
-    metadata_store_client: &MetadataStoreClient,
 ) -> anyhow::Result<GenericMetadataValue> {
-    let value: Option<GenericMetadataValue> = metadata_store_client
-        .get(ByteString::from(opts.key.as_str()))
-        .await?;
+    let value = super::get_value(connection, &opts.key).await?;
```

Comment:
We also access the metadata store from `remove_nodes` and `set_storage_state`.

Reply:
Ah sorry i missed those. Will do